apps/full-site-editing/bin/npm-run-build.js

```diff
@@ -26,6 +27,20 @@ const runOptions = {
 	printLabel: true,
 };
 
-runAll( commands, runOptions ).then( () => {
-	console.log( 'Finished running commands!' );
-} );
+runAll( commands, runOptions )
+	.then( () => {
+		console.log( 'Finished running commands!' );
+	} )
+	.catch( ( e ) => {
+		console.log( 'The build errored.' );
+		const tasks = e.results;
+		if ( Array.isArray( tasks ) ) {
+			const didNewspackSyncFail = tasks.some(
+				( task ) => task.name === 'build:newspack-blocks' && task.code !== 0
+			);
+			if ( didNewspackSyncFail ) {
+				console.log( 'You may need to run `composer install` from wp-calypso root.' );
+			}
```

Comment:
It would be nice to log `e.name` and `e.message` too, either with the initial error message or in an `else` here where we can't be more specific/helpful.

Props on the advice here, btw - I love it when an error points me in the right direction.

Reply:
The error object doesn't appear to include a name, but I've now added the message (which is just unhelpfully "build:newspack-blocks" exited with 1"). Still helpful to see which part of it failed!
  
<img width="1011" alt="Screen Shot 2020-07-22 at 12 55 58 PM" src="https://user-images.githubusercontent.com/6265975/88222422-d1d78580-cc1a-11ea-83ad-e2beb8c7ae33.png">